Header header
float32 arm_left
float32 arm_right
float32 bucket_left
float32 bucket_right
float32 sled
